### PR TITLE
Add missing Windows libraries

### PR DESCRIPTION
**Pull request by trembyle**
These two were skipped through .gitignore. I wasn't able to build until I added them back.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -39,5 +39,4 @@ xcuserdata/
 
 [Dd]ebug/
 [Rr]elease/
-x64/
-x86/
\ No newline at end of file
+x64/
```
Comment (kromenak) [inline .gitignore:42]: I think it makes sense to get rid of the x64 here, if we are getting rid of the x86.

File: Libraries/CopyLibraries.bat [hunk 2]
```diff
@@ -19,7 +19,7 @@ rem Copy SDL libraries.
 xcopy /y %ROOTDIR%\Libraries\SDL\lib\win\x86\SDL2.dll %OUTDIR%
 
 rem Copy Zlib libraries.
-xcopy /y %ROOTDIR%\Libraries\zlib\lib\win\x86\zlib.dll %OUTDIR%
+xcopy /y %ROOTDIR%\Libraries\zlib\lib\win\x86\zlib1.dll %OUTDIR%
```
Comment (kromenak) [inline Libraries/CopyLibraries.bat:22]: Hmm, I don't really like the name change here - was there a reason this was needed?

**Comment by kromenak** [comment]
Hi, thanks for catching this - I have these libraries on my end, but I did not realize they weren't being included due to gitignore lines.

I left a couple comments in the commit. I also think that there are some files included there that are not needed by the game (libz.a, libz.dll.a, zlib.def). I'd prefer to only include the absolute necessary files, which I think would just be "zlib.dll" and "zlib.lib".

As an alternative, I do have this all up and running on my end, so I'd be happy to update the gitignore on my end and commit these missing library files that I've been using for months.

File: Libraries/CopyLibraries.bat (hunk 2 above)
Comment (trembyle) [inline Libraries/CopyLibraries.bat:22]: The file is named `zlib1.dll` here:
http://gnuwin32.sourceforge.net/packages/zlib.htm

If I rename it to zlib.dll, the app does not start for me (can't unpack the resources).

Disregard the review thing. That was a misclick.

**Comment by trembyle** [comment]
> I also think that there are some files included there that are not needed by the game (libz.a, libz.dll.a, zlib.def). I'd prefer to only include the absolute necessary files, which I think would just be "zlib.dll" and "zlib.lib".

Done.

**Comment by kromenak** [comment]
Merged in! Thanks for the contribution.

I don't know too much about Windows DLLs, but I recently learned that the LIB file that accompanies the DLL is important. It is used at compile/link time so that the executable knows how to load the DLL at runtime.

The reason you can't change the name of the DLL is because the expected DLL name is embedded in the LIB file. So, at compile/link time, the executable takes the DLL name from the LIB file, and from that point, it can't easily be changed. And changing the name of the DLL in the LIB file is not exactly trivial.

Anyway, I thought about it a little more and realized...who cares what the DLL is called lol. Thanks again!

**Pull request merged**